src/semantize/typing.nit

```diff
@@ -1032,6 +1032,18 @@ redef class AExpr
 		end
 		return res
 	end
+
+	# Entry point for typing an expression.
```

Comment:
This is not really descriptive about what this method will do.

What about:

```
# Type the expression as if located in `visited_mpropdef`
```

Also, I wonder if this should be part of the public typing API? Should it be private and accessed through `intrude import`? Or maybe added by refinement later in the corresponding module?

In any case, better documentation will be useful.

Reply:
I think it's useful that the method for typing AExpr is part of the API in the same way the methods to type APropdef.

For the notion of public or private there is not really a better solution than the other. In our case defining the property as public does not seem problematic since the objective of the typing module are to provide the typing service so it seems logical that the entry point is public.

I changed the documentation a little, but it just refers to the documentation of visitors, because the provided method only uses them.